scripts/start-deployPufferfish

```diff
@@ -8,8 +8,8 @@ isDebugging && set -x
 
 IFS=$'\n\t'
 
-if [[ "${MAJOR_VANILLA_VERSION}" != "1.18" ]] && [[ "${MAJOR_VANILLA_VERSION}" != "1.17" ]]; then
-  log "ERROR: Pufferfish server type only supports versions 1.18 or 1.17, use PUFFERFISH_BUILD to select the the correct build 47 => 1.18.1, 50 => 1.18.2 etc"
+if [[ "${MAJOR_VANILLA_VERSION}" != "1.18" ]] && [[ "${MAJOR_VANILLA_VERSION}" != "1.17" ]] && [[ "${MAJOR_VANILLA_VERSION}" != "1.19" ]]; then
```

Comment:
I don't think I had the `versionLessThan` function before, so I wonder if this could now be:
```suggestion
if ! versionLessThan 1.17; then
```

Reply:
Using the `versionLessThan` function will accept string such as "1.18.2" while the download will fail because the `MAJOR_VANILLA_VERSION` should be `1.18` and `PUFFERFISH_BUILD` > 50. This may, in my opinion, mislead users (even I agreed that we should find a better solution than making a PR for each new major release).

Reply:
I'm not sure if your concern is actually applicable. Did you try out the suggestion and it didn't work?

Reply:
I think it would be at least good to sort them by version.